test/end-to-end/stock/stock.movements.spec.js

```diff
@@ -26,66 +26,52 @@ function StockMovementsRegistryTests() {
 
   const REFERENCE = 'SM.9.5';
 
-  it('finds lot for all time', async () => {
+  it('finds movements for all time', async () => {
     await modal.switchToDefaultFilterTab();
     await modal.setPeriod('allTime');
     await modal.submit();
-    await GU.expectRowCountAbove(gridId, 20);
+    await GU.expectRowCount(gridId, 16 + (2 * depotGroupingRow));
   });
 
   it('find entry movements ', async () => {
     // for Entry
     await modal.setEntryExit(0);
     await modal.switchToDefaultFilterTab();
-    await modal.setLimit(10);
     await modal.submit();
-    await GU.expectRowCount(gridId, 11); // count the grouping row
+    await GU.expectRowCount(gridId, 7 + (2 * depotGroupingRow));
   });
 
   it('filters by entry/exit', async () => {
     // for Exit
     await modal.setEntryExit(1);
     await modal.switchToDefaultFilterTab();
-    await modal.setLimit(100);
     await modal.submit();
-    await GU.expectRowCount(gridId, 12 + depotGroupingRow);
+    await GU.expectRowCount(gridId, 10 + depotGroupingRow);
   });
 
   it('find movements by depot', async () => {
     await modal.setDepot('Depot Secondaire');
     await modal.submit();
-    await GU.expectRowCount(gridId, 7);
+    await GU.expectRowCount(gridId, 2 + depotGroupingRow);
   });
 
   it('find movements by Service', async () => {
     await modal.setServiceUuid('Administration');
     await modal.submit();
-    await GU.expectRowCount(gridId, 2);
-  });
-
-  it('find movements by inventory', async () => {
```

Comment:
There was not any join with the inventory table so i was thinking that it was not necessary to have this test, since the registry was focused on just movement (stock_movement table) information